actors/builtin/multisig/multisig_test.go

```diff
@@ -1094,14 +1113,36 @@ func (h *msActorHarness) propose(rt *mock.Runtime, to addr.Address, value abi.To
 		Method: method,
 		Params: params,
 	}
-	rt.Call(h.a.Propose, proposeParams)
+	ret := rt.Call(h.a.Propose, proposeParams)
+	proposeReturn, ok := ret.(*multisig.ProposeReturn)
+	if !ok {
+		h.t.Fatalf("unexpected type returned from call to Propose")
+	}
+	if proposeReturn.Applied {
+		// get the last message sent and ensure it matches the exit code
+		msg := rt.LastMessageSent()
+		assert.Equal(h.t, proposeReturn.Code, msg.ExitCode())
+		var out runtime.CBORBytes
+		assert.NoError(h.t, msg.SendReturn().Into(&out))
+		assert.Equal(h.t, proposeReturn.Ret, out)
```

Comment:
What I really want to see is decoding into the result type for the method that was called, which depends entirely on which method that was and so can't be generically done here. You could pass a `CBORUnmarshaller` parameter to receive the return value.

Reply:
done in 622b3e2